internal/app/config/configs.go

```diff
@@ -106,7 +106,8 @@ func Dial(conf *MongoDB, logger glog.Logger) (*mongo.Client, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 100*time.Second)
 	defer cancel()
 	// Set client options
-	clientOptions := options.Client().ApplyURI(fmt.Sprintf("mongodb+srv://%s", conf.Address))
+	// clientOptions := options.Client().ApplyURI(fmt.Sprintf("mongodb+srv://%s", conf.Address))
+	clientOptions := options.Client().ApplyURI("mongodb://localhost:27017")
```

Comment:
You can test in local by updating the value in the config file. Btw, create a new config file config.local.yml